src/scene/vcFolder.cpp

```diff
@@ -342,7 +342,12 @@ void vcFolder::HandleImGui(vcState *pProgramState, size_t *pItemID)
         ImGui::Indent();
         ImGui::PushID(udTempStr("SXIExpanded%zu", *pItemID));
 
+        ImGui::BeginGroup();
         pSceneItem->HandleImGui(pProgramState, pItemID);
+        ImGui::EndGroup();
+
+        if (ImGui::IsItemClicked(0))
+          vcProject_SelectItem(pProgramState, m_pNode, pNode);
```

Comment:
I would probably recommend not doing it this way.

The majority the UI does not need to do the selection and this behaviour is not consistent with how other applications behave. The bug can be better fixed by showing the elements when the selected item is `-1`. Changing the speed of an attached model shouldn't select the item for example.

Reply:
So, you're suggesting just closing the PR and the AB item?

Reply:
or you want selection to open the treenode for an items handleimgui?